lib/index.js

```diff
@@ -51,7 +51,7 @@ export default function slackin({
   }
 
   let relativePath;
-  if (path.slice(-1) !== '/') {
+  if (!path.startsWith('/')) {
```

Comment:
We could probably use `path.isAbsolute()` but the current solution results in less changes. Let me know what you prefer.

Reply:
I think `startsWith` is fine for now—`path.isAbsolute` would make more sense semantically, but there's a lot of changes as it is, so let's limit the scope for now.